data/anaconda_options.txt

```diff
@@ -153,12 +153,6 @@ disklabel
 Prefer creation of the specified disk label type. Specify "gpt" to prefer creation of GPT disk
 labels. Specify "mbr" to prefer creation of MBR disk labels if supported.
 
-nodmraid
```

Comment:
I guess we could show a warning if `inst.nodmraid` is set. Please, look for `check_removed_arg "inst.nompath"` in our dracut code and add a new line for `inst.nodmraid`.

Reply:
Fixed.